pyproject.toml

```diff
@@ -24,10 +24,11 @@ maintainers =[
 ]
 description = """PECOS is a library/framework for the evaluation, study, and design of QEC protocols. It also provides the ability to study and evaluate the performance advanced hybrid quantum/classical compute execution models for NISQ algorithms and beyond."""
 readme = "README.md"
-requires-python = ">=3.9"
+requires-python = ">=3.10"
 license = { file = "LICENSE"}
 keywords = ["quantum", "QEC", "simulation", "PECOS"]
 dependencies = [
+    "phir~=0.1.6",
```

Comment:
FYI https://github.com/CQCL/phir/releases/tag/v0.2.0 now offers qparallel blocks. This PR may/may not want to support those. There is certainly interest in it, see #26

Reply:
Yeah, I was going to keep this to <0.2.0 until we bring PECOS up to 0.2.0 and deal with #26 in a separate PR.